code/game/objects/items/stacks/sheets/glass.dm

```diff
@@ -249,8 +249,8 @@
 	starting_materials = null
 
 /obj/item/stack/sheet/glass/rglass/recycle(var/datum/materials/rec)
-	rec.addAmount(MAT_GLASS, amount)
-	rec.addAmount(MAT_IRON,  0.5 * amount)
+	rec.addAmount(MAT_GLASS, 2750)

```

Comment:
* Hey nerd the recycling furnace counts 1 unit of materials as 1 sheet, so this will make duping materials **EVEN EASIER**.
* So with this, the recycled material doesn't get multiplied, so if I recycle 50 sheets of phazon I get 1 sheet back.